modules/ROOT/pages/associate-environments.adoc

```diff
@@ -5,15 +5,66 @@ endif::[]
 
 Before deploying applications or API gateways to Anypoint Runtime Fabric, you must associate a Runtime Fabric with one or more Anypoint Platform environments. When deploying a Mule application, you cannot select a Runtime Fabric as a deployment target until you have performed this step in the environment where you want to deploy.
 
+== Associate Business Groups
+
+You can share Runtime Fabric with an organization's business groups. This allows other business groups to deploy applications to a single Runtime Fabric. To use this feature, the business groups must fall under the scope of the organization the Runtime Fabric is registered to.
+
+To associate a Runtime Fabric across business groups:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select the *Runtime Fabric* tab on the left sidebar.
+. Select the Runtime Fabric to manage.
+. On the *Environments* tab, click *Add business group*.
```

Comment:
Correction:
On the Associated Environments tab, click Add business group.

Reply:
Based on a screenshot provided by the team, the UI tab shows as Environments